rsts/deployment/plugin_setup/webapi/snowflake.rst

```diff
@@ -0,0 +1,72 @@
+.. _deployment-plugin-setup-webapi-snowflake:
+
+Snowflake Plugin Setup
+----------------------
+
+This guide gives an overview of how to set up the Snowflake in your Flyte deployment.
+
```

Comment:
Do we also have to include the other two steps—cloning the Flytesnacks repo and starting the sandbox?

Reply:
yes

Reply:
some plugins will not work on the sandbox.

Reply:
Will Snowflake work or will it not work?

Reply:
Snowflake will work